cloudify_agent/installer/__init__.py

```diff
@@ -95,6 +95,13 @@ def _from_source(self):
         self.runner.run('virtualenv {0}'.format(
             self.cloudify_agent['envdir']))
 
+        if len(self.cloudify_agent['envdir']) > 110:
```

Comment:
110 feels quite arbitrary. In general I agree with the comment made on JIRA that relocatable should always be applied. My only concern here is with this code's behavior on windows. Let's remove the condition altogether, if the source installation test passes on appveyor, great, otherwise, let's apply this change on linux only